documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1,4 +1,8 @@
-= Debezium Connector for PostgreSQL
+
+[id="debezium-connector-for-postgresql"]
+= {ProductName} Connector for PostgreSQL
```

Comment:
This is the same problem as the mongodb.adoc headings with attribute substitution.